odoo/addons/base/ir/ir_qweb/fields.py

```diff
@@ -367,8 +367,9 @@ class FloatTimeConverter(models.AbstractModel):
 
     @api.model
     def value_to_html(self, value, options):
-        hours, minutes = divmod(value * 60, 60)
-        return '%02d:%02d' % (hours, minutes)
+        sign = 1 if value >= 0 else -1
```

Comment:
i'd have thought python would have something nicer to get the sign of a number, couldn't find anything

Reply:
Hello @dbo-odoo 
We can use this 
sign  = int(math.copysign(1, value))